apps/src/templates/manageStudents/ManageStudentsTable.jsx

```diff
@@ -618,6 +624,12 @@ class ManageStudentsTable extends Component {
     window.open(url, '_blank');
   };
 
+  showSectionCodeDialog = () => {
```

Comment:
See [comment about binding](https://github.com/code-dot-org/code-dot-org/pull/37209/files#r504330031) below. Given that this is matching the style in this component, it's ok to leave it this way, but I'd advocate for migrating all bound functions in this component into the constructor while you're here.

Reply:
Just to clarify, I only need to bind the function context in the constructor if `this` is referenced within the body of the function. For example, the function `isEditingDisabled` is:
```
isEditingDisable = userType => {
    return userType === ‘teacher’;
}
```
I can just change this to `isEditingDisabled(userType)` without binding in the constructor, correct?